docs/2_step_by_step/2_7_get_tuned_model.md

```diff
@@ -1 +1,47 @@
-# Retrieve tuned model
\ No newline at end of file
+# Retrieve tuned model
+
+Perfect!
+Now you have started the fine-tuning job in the cloud.
+However, fine-tuning take times,
+it highly depends on the size of your training data, evaluation data and other hyper-parameters.
+
+Once fine-tuning is ready,
+you can get the tuned model with:
+
+```python
+import finetuner
+from docarray import DocumentArray
+
+finetuner.login()
+
+train_data = DocumentArray(...)
+
+run = finetuner.fit(
+    model='efficientnet_b0',
+    train_data=train_data,
+)
```

Comment:
And if we have an example with non-default experiment above, let's show here how developers can reconnect to finetuner and retrieve the run (since this will happen quite frequently).

```python
experiment = finetuner.get_experiment('finetune-flickr-dataset')
run = experiment.get_run('finetune-flickr-dataset-efficientnet-1')
```